### ISSUE TITLE
A5 paper size

### ISSUE THREAD

**Issue by yigithsyn** [nobuhito/vscode.printcode#9]
Could you please add settings option for A5 paper size?

```
"printcode.paperSize": {
	"type": "string",
	"description": "Paper size and orientation.",
	"enum": [
		"a4",
		"a4Land",
		"letter",
		"letterLand",
		"a5"
	],
	"default": "a4"
}
```

Thanks.

**Comment by janilahti** [comment]
It's easy to add A5, and maybe legal as well while we're onto it.

My previous fix, PR #7 , has gone unnoticed by extension author for over two weeks now, so I'm hesitant to submit more PRs if they don't get merged and included in version available in vscode marketplace.

**Comment by yigithsyn** [comment]
Thank you very much. Really appreciated!
